adminguide.rst

```diff
@@ -320,6 +322,52 @@ job queue and record of past jobs.
     If this happens it is best to stop Flux, remove the content.sqlite file,
     and restart.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+flux-accounting configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to support fairshare and multi-factor priority ordering of user jobs, the flux-accounting package must be present and configured for the system instance. This allows for the management of users and banks on a cluster, job priorities to be calculated using multiple factors and uses job record information from users to calculate and maintain fairshare values.
+
+The first thing to configure when first setting up the flux-accounting database is to set the PriorityUsageResetPeriod and PriorityDecayHalfLife parameters. Both of these parameters represent a number of weeks by which to hold usage factors up to the time period where jobs no longer play a factor in calculating a usage factor. If these parameters are not passed when creating the DB, PriorityDecayHalfLife is set to 1 week and PriorityUsageResetPeriod is set to 4 weeks, i.e the flux-accounting database will store up to a month's worth of jobs broken up into one week chunks:
+
+.. code-block:: console
+
+ $ flux account create-db --priority-decay-half-life=2 --priority-usage-reset-period=8 path/to/DB
```

Comment:
What node should this run on?  The node designated as the flux rank 0 node?   Are there storage requirements that need to be mentioned?

Reply:
BTW, see storage config above.  We've already set up `/var/lib/flux` to be owned by user flux and have plenty of space.  Maybe that is where the db should be?   We could replace /path/to/db with that in the provided commands for easy cut and paste.

Reply:
I'm sorry in advance if this is a dumb answer, as I'm still new to cluster management and whatnot, but I _believe_ the database should be located on the management node - is this the node designated as the flux rank 0 node? The only storage requirements that I can think of is just enough space to hold a sqlite file that could holds all of the flux-accounting information - users, banks, shares, etc.

> We've already set up /var/lib/flux to be owned by user flux and have plenty of space. Maybe that is where the db should be? We could replace /path/to/db with that in the provided commands for easy cut and paste.

I like that suggestion. I can make that change to that example command.

Reply:
Yes, the rank 0 node is how we referred to it in the storage section.   So for consistency you could use the same language here and put management node in parenthesis if you like :-)